src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -2792,6 +2797,18 @@ bool Creature::LoadCreaturesAddon(bool reload)
     return true;
 }
 
+void Creature::LoadSparringPct()
+{
+    ObjectGuid::LowType spawnId = GetSpawnId();
+    auto const& sparringData = sObjectMgr->GetSparringData();
+
+    auto itr = sparringData.find(spawnId);
+    if (itr != sparringData.end() && !itr->second.empty())
+    {
+        _sparringPct = itr->second[0];
```

Comment:
Need select random element from container instead of first

Reply:
Why ? One SpawnId, one pct value ?